x/blob/client/cli/payforblob.go

```diff
@@ -31,19 +32,32 @@ const (
 
 func CmdPayForBlob() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "PayForBlobs [hexNamespaceID] [hexBlob]",
-		Short: "Pay for a data blob to be published to the Celestia blockchain",
-		Long: "Pay for a data blob to be published to the Celestia blockchain. " +
-			"[hexNamespaceID] must be a 10 byte hex encoded namespace ID, " +
-			"(for the namespace version 0). " +
-			"[hexBlob] can be an arbitrary length hex encoded data blob. " +
-			"Hex strings should not include a 0x prefix. " +
-			"This command only supports a single blob per invocation. ",
-		Args: cobra.ExactArgs(2),
+		Use: "PayForBlobs namespaceID blob",
+		// This example command can be run in a new terminal after running single-node.sh
+		Example: "celestia-appd tx blob PayForBlobs 0x00010203040506070809 0x48656c6c6f2c20576f726c6421 \\\n" +
+			"\t--chain-id private \\\n" +
+			"\t--from validator \\\n" +
+			"\t--keyring-backend test \\\n" +
+			"\t--fees 21000utia \\\n" +
+			"\t--yes",
```

Comment:
```
  -y, --yes                       Skip tx broadcasting prompt confirmation
```
looks like there isn't an alias for `--yes` to `--skip-confirmation` so this requires an unrelated code change. Will create a new issue for it. Pretty sure this is something we would do upstream in Cosmos SDK rather than in celestia-app.

Reply:
https://github.com/celestiaorg/celestia-app/issues/2433